src/tests/ftest/util/test_utils_container.py

```diff
@@ -302,6 +306,14 @@ def create(self, uuid=None):
         kwargs = {"poh": self.pool.pool.handle}
         if uuid is not None:
             kwargs["con_uuid"] = uuid
+        # Refer daos_api for setting input params for DaosContainer.
+        if con_in is not None:
+            self.input_params.type = con_in[0]
+            self.input_params.enable_chksum = con_in[1]
```

Comment:
should we use constants here?   Seems error prone to use numeric indexes.

Reply:
con_in is the array which has constant values read from the yaml file.

Reply:
that seems like it needs some rework.  You can add keys in a yaml file.   Having a list relies on the user getting it in the right order, no?

Reply:
Yes, if we need to modify now, yes it will need some rework. The user needs to know the sequence. I have tried to add good comment on the list structure. We are trying the same for ior flags also. Tried to maintain the same method instead of having several keys for different container properties.  Johann wanted initially to pack several container properties into a structure (daos_api.py) rather than having several input parameters.  This method will be helpful as we increase more options later.

Reply:
I have no problem with packing data.  I just don't like numeric constants used to identify it.   It would be better to do something like define some constants and use them instead

Reply:
Sure. As discussed, I will do the changes in a separate PR.